modules/kubernetes_node_pools/main.tf

```diff
@@ -1,59 +1,100 @@
 locals {
   default_labels = {
-    environment = "${var.environment}"
+    environment = var.environment
   }
 }
 
 resource "google_container_node_pool" "node_pool" {
-  name               = "${lookup(var.node_pools[count.index], "name")}"
-  count              = "${var.regional_cluster ? 0 : length(var.node_pools)}"
+  name               = var.node_pools[count.index]["name"]
+  count              = var.regional_cluster ? 0 : length(var.node_pools)
```

Comment:
Shouldn't we use if instead of count with 0.12?

Reply:
As above :)